pytest_jupyterhub/jupyterhub_spawners.py

```diff
@@ -3,9 +3,11 @@
 can be used to test different spawner implementations
 """
 
-from pytest import fixture
+import pytest
+from jupyterhub.spawner import Spawner
 
 
-@fixture
-def app_for_spawners():
-    pass
+@pytest.fixture
+def spawner_class():
+    # returns the spawner class to be used by the hub
```

Comment:
Let's write the comment about the fixture as a docstring so that we can later autogenarate the docs using https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html and https://github.com/jupyterhub/autodoc-traits.

```suggestion
    """Allows tests to setup the Spawner class that will be used by the hub."""
```